packages/qtile-git/qtile-git.pacscript

```diff
@@ -0,0 +1,39 @@
+name="qtile-git"
+pkgname="qtile"
+url="https://github.com/qtile/qtile.git"
+license="MIT"
+depends="python3-cffi xserver-xorg python3-xcffib python3-cairocffi libpangocairo-1.0-0 python3-pip python3-dbus libpulse-dev pavucontrol"
+description="A full-featured, hackable tiling window manager written and configured in Python"
+maintainer="Ruturajn <nanotiruturaj@gmail.com>"
+
+pkgver() {
+  git ls-remote "${url}" master | cut -f1
+}
+version="$(pkgver)"
+
+prepare() {
+  true
+}
+
+build() {
+  true
+}
+
+install() {
+  pip install --upgrade --no-cache --force-reinstall cairocffi
+  pip3 install dbus-next
+  pip3 install .
+  sudo python3 setup.py install
+  sudo mkdir -p /usr/src/pacstall/qtile-git/usr/share/xsessions
+  echo "[Desktop Entry]
+  Name=Qtile
+  Comment=Qtile Session
+  Exec=qtile start
+  Type=Application
+  Keywords=wm;tiling" | sudo tee -a /usr/src/pacstall/qtile-git/usr/share/xsessions/qtile.desktop > /dev/null
```

Comment:
@Henryws shouldn't this be a separate file though?

Reply:
Henryws: https://discord.com/channels/839818021207801878/866673569521991700/956582537273176105

Reply:
I am not exactly sure, what you mean here

Reply:
> I am not exactly sure, what you mean here

@Henryws replied on discord that it's fine to have it embedded in the pacscript. That's the discord link to the message for archival purposes.